Sources/Views/Collections/CollectionSection.swift

```diff
@@ -197,21 +199,20 @@ private func relatedItem(_ relatedItem: Episode.Collection.Section.Related) -> N
           Class.flex.items.center,
           Class.grid.row,
           Class.margin([
-            .desktop: [.top: 2, .bottom: 3],
-            .mobile: [.top: 1, .bottom: 2],
+            .desktop: [.top: 2],
+            .mobile: [.top: 1],
           ]),
           Class.padding([.mobile: [.leftRight: 2]]),
           Class.pf.collections.hoverBackground,
           Class.pf.collections.hoverLink,
           Class.pf.colors.border.gray800,
           Class.pf.colors.bg.gray900,
         ]),
-        .href(url(to: .episode(.show(.right(episode.id))))),
+        .href(url(to: .episode(.show(.left(episode.slug))))),
         .style(
-          backgroundColor(.other("#fafafa"))
-          <> borderColor(all: .other("#e8e8e8"))
-          <> borderWidth(left: .px(4))
-          <> height(.px(48))
+          borderColor(all: .other("#e8e8e8"))
```

Comment:
removed the background color inline style so that it didn't override the hover.

I think we should move off of inline styles as much as possible. it's hard to do when you need to give a little bump to something (like the 4px margin between lessons), but other times we could either pick something as close as possible to what is already in our style guide, or we could introduce new utility classes.

Reply:
I also removed the explicit `height: 48px` and instead relied on the internal padding of the row to give it its height